React_pakadmin/tar_19/src/SortableTable.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import "../css/main.css";
+
+export default function SortableTable({ data }) {
+  const columnsEnum = { name: 1, country: 2, email: 3 };
+  const [inputData, setInputData] = useState(data.slice(1, data.length));
+  const [columnsClickCounter, setColmnsClickCounter] = useState({
+    name: 0,
+    country: 0,
+    email: 0,
+  });
+
+  function sortInputData(columnName) {
+    setColmnsClickCounter({
+      ...columnsClickCounter,
+      [columnName]: columnsClickCounter[columnName] + 1,
+    });
+    const results = inputData.sort(function (a, b) {
+      if (a[columnsEnum[columnName]] > b[columnsEnum[columnName]]) {
```

Comment:
This is why you **never** use one-letter variables. Here they stop making sense and the whole phrase is much less readable. `row1` and `row2` for example, would make much more sense.